nav2_costmap_2d/costmap_plugins.xml

```diff
@@ -22,5 +22,12 @@
       <description>Prevents the robot from appearing in keepout zones marked on map.</description>
     </class>
   </library>
+
+  <library path="nav2_3d_static_layer">
+    <class type="nav2_costmap_2d::Nav23dStaticLayer"  base_class_type="nav2_costmap_2d::Layer">
```

Comment:
Lets make this name a little easier to understand: how's `StaticLayer3D`?

Reply:
`StaticLayer3D` is much better than `Nav23dStaticLayer`, I have changed the layer name.